src/components/DataView/DataViewTBody.tsx

```diff
@@ -2,14 +2,15 @@ import React from "react";
 import styled from "styled-components";
 import jsvalidator from "jsvalidator";
 import { DragDropContext, Droppable, DropResult } from "react-beautiful-dnd";
-import DataViewTr from "./DataViewTr";
+import DataViewTr from "./DataViewTr/DataViewTr";
 import theme from "@root/theme";
 import { MosaicObject } from "@root/types";
 // import { DataViewAction, DataViewAdditionalAction } from "./DataViewTypes";
 
 const StyledTBody = styled.tbody`
 	& > tr {
 		border-bottom: ${theme.borders.lightGray};
+		padding: 8px 12px;

```

Comment:
I don't think the 8px padding should go here, if you use the inspect element over any td it'll show this: 
![image](https://user-images.githubusercontent.com/23409284/203443380-44438055-90f2-4a1a-a326-085f60e50fc6.png)

This is giving each element 5px of padding (this is a few lines under this). Also the 12px should only be added to the left of the first td and the right of the last td